widget/toolbar_test.go

```diff
@@ -74,6 +74,15 @@ func TestToolbar_ItemPositioning(t *testing.T) {
 	}
 }
 
+func TestToolbar_SetIcon(t *testing.T) {
+	oldIcon := theme.FyneLogo()
+	toolbarItem := NewToolbarAction(oldIcon, func() {})
```

Comment:
While not a deal-breaker, I just want to point out that you can just pass `nil` instead of an empty function.